src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -142,6 +143,16 @@ public function setRequired($boolean)
     }
 
     /**
+     * Set this node as deprecated.
+     *
```

Comment:
You should hint here that `%node%` and `%path%` have a special meaning in a deprecated message.